### ISSUE TITLE
Random to raise ValueError for negative probabilities

### ISSUE THREAD

**Issue by taless474** [STEllAR-GROUP/phylanx#891]
The following is an implementation of [Keras dropout](https://keras.io/backend/#dropout) which NumPy implementation is [here](https://github.com/STEllAR-GROUP/keras/blob/master/keras/backend/numpy_backend.py#L607):
```py
import numpy as np
from phylanx import Phylanx, PhylanxSession, execution_tree
import pytest

PhylanxSession.init(1)
	
def variable(value, dtype=None, name=None, constraint=None):
    return execution_tree.var(np.array(value, dtype))
	
def learning_phase():
	return True
	
@Phylanx
def _dropout(x, level, noise_shape, seed):
	if seed:
		set_seed(seed)
	noise = random(noise_shape if noise_shape else shape(x), ["bernoulli", 1 - level])
	return x * noise / (1-level)
	
@Phylanx
def _no_dropout(x):
	return x

def dropout_eager(x, level, noise_shape, seed):
	if learning_phase():
		return _dropout.lazy(x, level, noise_shape, seed)

	# dropout is not considered outside the learning phase
	return _no_dropout.lazy(x)

def dropout(x, level, noise_shape=None, seed=None):
	return dropout_eager(x, level, noise_shape, seed)
	
def eval(func):
    return func.eval()
	
def test_dropout(val):		
	# Test invalid use cases
	with pytest.raises(ValueError):
		z = dropout(variable(val), level=-0.5)

val = np.random.random((2, 5))
print(test_dropout(val))
```
We need to raise a ValueError for a negative `level`. In the PhySL code of:
```console
random(make_list(2,5),list("bernoulli",-.5))
```
I get

![1](https://user-images.githubusercontent.com/20496946/54765874-46825600-4bc8-11e9-8619-81bae7a732ec.png)
